CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 CHANGELOG
 =========
 
-## HEAD (Unreleased)
+## 1.4.0 (2022-02-02)
 
 - Default namespace for resources that don't provide one is now the service accounts namespace (where the operator is likely running) instead of "default"
+  Note - this is a potential breaking change. To revert to the previous behavior unset `PULUMI_INFER_NAMESPACE` in the operator pod spec environment variables
+  (https://github.com/pulumi/pulumi-kubernetes-operator/pull/249/)
```

Comment:
May want to highlight this separately from the other bullets and include an example

Reply:
Updated. Thoughts?